Week1/homework/js-exercises/ex3.js

```diff
@@ -0,0 +1,14 @@
+//Exercise 3: Log the number
+//declare your variable numberX
+let numberX;
```

Comment:
nice that you're already using let and const!

Reply:
it's really was just var when i start the code, but i remember Wilgert when he say don't use var in the class, i need to remember that with every declaration.